ui/components/multichain/create-eth-account/create-eth-account.js

```diff
@@ -7,7 +7,7 @@ import {
   setAccountLabel,
   getNextAvailableAccountName as getNextAvailableAccountNameFromController,
 } from '../../../store/actions';
-import { CreateAccount } from '..';
+import { CreateAccount } from '../create-account';
```

Comment:
?

Reply:
It was causing some test breaking, the import couldn't be read otherwise.